lib/siwapp_web/live/customer_live/edit.ex

```diff
@@ -1,8 +1,9 @@
-defmodule SiwappWeb.CustomerLive.Edit do
+defmodule SiwappWeb.CustomersLive.Edit do
```

Comment:
Do not change this file at all. This is not your business in this PR

Reply:
Okay, just trying to help there